### ISSUE TITLE
When changing the field type, the x and y coordinates are changed to 0.

### ISSUE THREAD

**Issue by hand-dot** [pdfme/pdfme#513]
### Describe the bug

![CleanShot 2024-07-23 at 10 04 50](https://github.com/user-attachments/assets/ceab9552-beec-4d97-b81c-88abd7d32930)

This must have happened due to recent changes. It only occurs in the local playground and does not occur in the [production playground](https://playground.pdfme.com/).

### To Reproduce

Edit Field -> Change the field type
![CleanShot 2024-07-23 at 10 11 00@2x](https://github.com/user-attachments/assets/183372e6-7ff3-46d6-941a-8e037097ebcc)

### Expected behavior

Do not change the position.

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui): @pdfme/ui
- pdfme version: latest
- Operating system: Mac
- Node.js version or Browser name & version: latest chrome
```

### Your Error Log

```shell
none
```

**Comment by hand-dot** [comment]
The following commit is likely related to this issue.

REF: https://github.com/pdfme/pdfme/commit/1008ff816a78828544075d0811e24dc37ba6791e

**Comment by peteward** [comment]
I have a further fix for the watch command handling a few cases, but it does not seem to be the reason why it's setting x and y to 0. I will continue investigating...

**Comment by hand-dot** [comment]
@peteward Thank you always. I will release this soon.

**Comment by hand-dot** [comment]
Released as 4.2.3!
